src/persist-client/src/internal/metrics.rs

```diff
@@ -71,8 +72,20 @@ pub struct Metrics {
 
 impl Metrics {
     /// Returns a new [Metrics] instance connected to the given registry.
-    pub fn new(registry: &MetricsRegistry) -> Self {
+    pub fn new(cfg: &PersistConfig, registry: &MetricsRegistry) -> Self {
         let vecs = MetricsVecs::new(registry);
+        let start = Instant::now();
+        let uptime = registry.register_computed_gauge(
+            metric!(
+                name: "mz_persist_metadata_seconds",
+                help: "server uptime, labels are build metadata",
```

Comment:
is anything here persist-specific? seems like all generally useful information

Reply:
nope! i linked this in the commit message, but this is modeled after https://github.com/MaterializeInc/materialize/blob/f5a7db6309c4d8d990ab4bf12e8f128a26bc8c0b/src/materialized/src/server_metrics.rs#L55-L79. i tried to resurrect the general one a couple weeks ago, but there just isn't a good place to put it right now; I spent a couple hours poking and was unsuccessful. happy to rip this out later if someone does manage to add it, but i've wanted this several times now, so I think it's valuable to get this version merged in the meantime